hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieHFileReader.java

```diff
@@ -250,7 +259,7 @@ public BloomFilter readBloomFilter() {
    */
   public List<Pair<String, R>> readRecords(List<String> keys, Schema schema) throws IOException {
     this.schema = schema;
-    reader.loadFileInfo();
+    reader.getHFileInfo();
```

Comment:
From what i see those are returning cached `fileInfo` object do we really need those here?

Reply:
I think it's just to keep the behavior consistent. We can probably remove it. Also, this method does not exist in hbase 2.2.3.

Reply:
@yihua are you planning on removing them?